packages/react-components/src/components/VideoGallery/FloatingLocalVideoLayout.tsx

```diff
@@ -208,7 +208,7 @@ export const FloatingLocalVideoLayout = (props: FloatingLocalVideoLayoutProps):
         isShort={isShort}
         onFetchTilesToRender={setIndexesToRender}
         isNarrow={isNarrow}
-        shouldFloatLocalVideo={true}
+        shouldFloatLocalVideo={!!localVideoComponent}
```

Comment:
Is `localVideoComponent` valid when the video is turned off but the local video avatar shows in the floating form?

Reply:
Yup. It is valid when the local video stream is turned off.